src/app/component/ProductGallery/ProductGallery.component.js

```diff
@@ -68,13 +87,16 @@ class ProductGallery extends Component {
 
     render() {
         const gallery = this.getGalleryPictures();
+        const { activeImage } = this.state;
 
         return (
             <div block="ProductGallery">
                 { this.renderAdditionalPictures(gallery) }
                 <Slider
                   mix={ { block: 'ProductGallery', elem: 'Slider' } }
                   showCrumbs
+                  activeImage={ activeImage }
+                  changeParentActiveImage={ i => this.changeActiveImage(i) }
```

Comment:
I believe you can bind function in constructor and change this line to `changeParentActiveImage={ this.changeActiveImage }`